devops/ci/multi_node_deploy/Jenkinsfile_3Node_Deploy

```diff
@@ -155,7 +166,7 @@ node {
                 sshCommand remote: remotes[0], command: """
                     yum install -y sshpass
                     export SSHPASS=${PASSWORD}
```

Comment:
not related to the change but ... for such cases makes sense to consider [withCredentials](https://www.jenkins.io/doc/pipeline/steps/credentials-binding/)

Reply:
We want the password to be passed as an argument from Build Parameters. WithCredentials might help us hiding the password. In fact, so can the https://plugins.jenkins.io/mask-passwords/ plugin.
Let me check if we can have that.